src/renderer/renderer.ts

```diff
@@ -806,7 +804,7 @@ export class SDFGRenderer {
             this.canvas_manager.set_user_transform(user_transform);
 
         // Resize event for container
-        const observer = new MutationObserver((mutations) => {
+        const observer = new MutationObserver(() => {
```

Comment:
are you sure about this?

Reply:
The argument isn't being used and the call signature accepts 0-2 arguments according to the docs.